docs/docs/aws.md

```diff
@@ -385,6 +385,7 @@ Note, the path resolution logic for `ObjectStoreLocationProvider` is `write.data
 However, for the older versions up to 0.12.0, the logic is as follows:
 - before 0.12.0, `write.object-storage.path` must be set.
 - at 0.12.0, `write.object-storage.path` then `write.folder-storage.path` then `<tableLocation>/data`.
+- at 2.0.0 `write.object-storage.path` and `write.folder-storage.path` will be removed
```

Comment:
it seems that the existing line 387 above missed one config `write.data.path`. 

or maybe `write.data.path` was added after 0.12.0. then we missed a line/version altogether.

Reply:
This also confused me, but the trick is on line 384 (that just falls of in this diff:

> Note, the path resolution logic for `ObjectStoreLocationProvider` is `write.data.path` then `<tableLocation>/data`.